server/config/config.go

```diff
@@ -23,6 +23,9 @@ const (
 
 	// DefaultGRPCWebAddress defines the default address to bind the gRPC-web server to.
 	DefaultGRPCWebAddress = "0.0.0.0:9091"
+
+	// DefaultChanCheckTxSize defines the default maximum size of channel check tx in Baseapp
```

Comment:
Do we need the word "maximum" here? It may sound like, "you can set the size of it, but the maximum would be this value".
```suggestion
	// DefaultChanCheckTxSize defines the default size of channel check tx in Baseapp
```

Reply:
Yes, this is what I want. Because this is the maximum value and this is numbers greater than the set value are not assigned.

Reply:
Oh!, I see. Since the `DefaultChanCheckTxSize` value is a default value, so you're saying it's best not to set to the highest value?